src/libraries/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests.csproj

```diff
@@ -68,5 +68,7 @@
       <OutputItemType>Content</OutputItemType>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </ProjectReference>
+    <!-- R2R testing does not tolerate the combination of a regular project reference and a content reference. -->
+    <PublishReadyToRunExclude Include="LongPath.dll" />
```

Comment:
Curious, why does R2R testing not allow a ProjectReference that contributes to the `Content` item?